sdk/search/search-documents/review/search-documents.api.md

```diff
@@ -163,6 +175,13 @@ export type BM25Similarity = Similarity & {
     b?: number | null;
 };
 
+// @public
+export interface CaptionResult {
+    [property: string]: any;
+    readonly highlights?: string | null;
```

Comment:
This null is being handled by Issue https://github.com/Azure/autorest.typescript/issues/909 which will be changed soon.